app/src/main/java/com/jalasoft/routesapp/ui/home/viewModel/HomeViewModel.kt

```diff
@@ -88,4 +95,71 @@ constructor(private val placeManager: PlaceRepository, private val gDirectionsRe
     fun deleteFavoriteDestination(favoriteDestinationEntity: FavoriteDestinationEntity) {
         localDB.deleteFavoriteDestination(favoriteDestinationEntity)
     }
+
+    fun checkForUpdatedData(context: Context) = viewModelScope.launch {
```

Comment:
doesn't this should be added on their own Manager or repository? as they're not related to the viewmodel as is

Reply:
Did you mean that this doesn't have to be in the viewmodel or that i have to create an own repositoty and manager for this instead of using the other repositories? because i'm not sure if this shouldn't be in a viewmodel, but i put these functions in another thread so it won't freeze the UI while doing the sync

Reply:
done